### ISSUE TITLE
Right-click alternative for touchscreen

### ISSUE THREAD

**Issue by nas00** [Komorebi-Fork/komorebi#28]
Is there a way to get to launch the wallpaper selection menu without a mouse right-click?

**Comment by Kangie** [comment]
After a quick browse through the code, I think that it's pretty specifically looking for the 'Right click' button press [here](https://github.com/Komorebi-Fork/komorebi/blob/94faabf60983ed1dbd0a5e2591f943ae76dabea5/src/OnScreen/BackgroundWindow.vala#L197).

For touchscreen, would you prefer a 'long press' action to open the existing menu (in addition to right-clicking), or an easy way to access the 'Wallpaper Selector'?

**Comment by nas00** [comment]
A menu entry or command line like the wallpaper creator would be easiest. I don't believe long press is supported on my device.

**Comment by Kangie** [comment]
Vala Documentation [suggests](https://valadoc.org/gtk+-3.0/Gtk.GestureLongPress.html) that we should be able to handle the 'long press' at the application level.

@Ev1lbl0w, any thoughts on implementation?

**Comment by Ev1lbl0w** [comment]
Yes, I think making the configuration menu separate is important (#17); my argument is that it might be possible for an invalid config to prevent Komorebi from launching, and then the user has no choice but to uninstall and/or edit configs by hand.

As to implementation, I'd say the thoughest part is how will the configuration menu communicate with the main app when they become separated. I guess the time to learn [D-Bus](https://www.freedesktop.org/wiki/Software/dbus/) has finally come :stuck_out_tongue: 

I can take a shot at this, but only after I finally finish my work on porting the code to Python *(I'd say it's 80% done)*
